system-int-test/test/cmr/system_int_test/search/generic_association_test.clj

```diff
@@ -45,6 +45,216 @@
 ;; Tests
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 
+;; Test that generic associations can be made between generic documents and tools.
```

Comment:
This test also tested association with collections via the old endpoint.

Reply:
Done.